### PR TITLE
Fix - TRSV2-498 - ROI timeout issue

### PR DESCRIPTION
**Pull request by chris-pettinga**

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -34,7 +34,7 @@ repos:
         additional_dependencies: [flake8==5.0.4]
   - repo: https://github.com/uktrade/fitness-functions
     fail_fast: true
-    rev: a0895d95d9ca3b41a1a18e5fee5348a77df36f8f
+    rev: '1.1'
```
Comment (nahmed99) [inline .pre-commit-config.yaml:37]: Just double-checking that this replacement is correct?

File: .pre-commit-config.yaml [hunk 2]
```diff
@@ -43,14 +43,14 @@ repos:
...
-    rev: v4.3.0
+    rev: v4.4.0
     hooks:
       - id: mixed-line-ending
         name: Replaces line endings with the most commonly used one
       - id: detect-private-key
         name: Detects for the presence of private keys in the code
   - repo: https://github.com/rtts/djhtml
-    rev: b26c4306e3676521961e69675afb31bea60a7110
+    rev: v2.0.0
```
Comment (nahmed99) [inline .pre-commit-config.yaml:53]: Just double-checking that this replacement is correct?

File: .pre-commit-config.yaml (hunk 1 above)
Comment (chris-pettinga) [reply .pre-commit-config.yaml:37]: yes, automatic update with pre-commit finds the latest revision from github and replaces it here

File: .pre-commit-config.yaml (hunk 2 above)
Comment (chris-pettinga) [reply .pre-commit-config.yaml:53]: same as above

**Review by nahmed99 — APPROVED** [review]

**Review by osimuka — APPROVED** [review]

**Pull request merged**